src/components/checkbox/checkbox.style.ts

```diff
@@ -132,6 +132,7 @@ const StyledCheckbox = styled.div<StyledCheckboxProps>`
       svg {
         height: 24px;
```

Comment:
comment (non-blocking): we could use the `--sizing300` token here instead of hard-coding 24px, don't mind if you leave it but if you do update it's worth doing it for the whole file wherever possible

Reply:
Thinking about this some more, lets just update the file to use tokens in a new ticket as there are other style files for this component